src/main/java/io/github/townyadvanced/townycombat/tasks/JumpReductionTask.java

```diff
@@ -0,0 +1,53 @@
+package io.github.townyadvanced.townycombat.tasks;
+
+import com.palmergames.bukkit.towny.Towny;
```

Comment:
The suggestion below will mean this import has to change.

Reply:
Done